Sources/App/Core/Search+Result.swift

```diff
@@ -0,0 +1,94 @@
+extension Search {
+    enum Result: Codable, Equatable {
```

Comment:
NB: this is currently relying on Swift 5.5 Codable auto synthesis and will not compile on Swift 5.4 (hence the CI failure).

We can either add manual conformance or move to Swift 5.5 before (not as part of) this PR.

Reply:
I've added manual conformance now, with a deprecation warning when using 5.5.